recipes/concurrencpp/all/conanfile.py

```diff
@@ -57,7 +57,7 @@ def layout(self):
     def validate(self):
         if self.settings.compiler.get_safe("cppstd"):
             check_min_cppstd(self, self._min_cppstd)
-        if self.options.shared and is_msvc(self):
+        if Version(self.version) <= "0.1.5" and self.options.shared and is_msvc(self):
```

Comment:
https://github.com/David-Haim/concurrencpp/issues/75 has been fixed in 0.1.6, thus shared builds on Windows are supported now.

Reply:
Nice, do you have any reference/issue/url indicating that fix? It would be great for traceability.

Reply:
Yes, this is the MR where it got fixed: https://github.com/David-Haim/concurrencpp/pull/94

Reply:
@chausner Thank you!